frontend/src/components/SideBar/SideBar.tsx

```diff
@@ -5,16 +5,24 @@ import feedIcon from "../../assets/feedIcon.svg";
 import historyIcon from "../../assets/historyIcon.svg";
 import logoutIcon from "../../assets/logoutIcon.svg";
 import { Link } from "react-router-dom";
+import { useEffect, useState } from "react";
 
 const SideBar = () => {
+  const [username, setUsername] = useState("");
```

Comment:
nit: is there a way to pass this down via context from the root component of the app? Ideally there is some sort of currentUser global in the app:

eg. 
```
const {currentUser} = useContext(CurrentUserProvider)`;
const username = currentUser.username
```

Reply:
I'm about to work on this in my next PR